website/client/components/groups/questDetailsModal.vue

```diff
@@ -180,6 +180,7 @@ export default {
       return quests.quests[this.group.quest.key];
     },
     members () {
+      let party = this.partyMembers || [];
```

Comment:
I think this should be `let partyMembers = this.partyMembers || [];`and then use `partyMembers` instead of `this.partyMembers` below